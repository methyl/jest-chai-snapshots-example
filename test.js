
describe("meaning of the world", () => {
  it("equals to 42", () => {
    expect(42).to.equal(42)
  })

  it("equals to 42 using snapshots", () => {
    expect(42).to.matchSnapshot()
  })
})
