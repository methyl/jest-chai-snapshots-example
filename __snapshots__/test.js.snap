// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`meaning of the world equals to 42 using snapshots 1`] = `undefined`;
